test/removeLast.test.js

```diff
@@ -0,0 +1,13 @@
+const { removeLast } = require("../lib");
+
+test("duck is removed from array", () => {
+  expect(removeLast(["cat", "dog", "fish", "duck"])).toBe(["cat", "dog", "fish"]);
```

Comment:
These tests should use toEqual, as toBe checks for same identity. Please adapt this so we can merge :)